dashboard/models.py

```diff
@@ -197,24 +199,28 @@ class Meta:
         db_table = 'course_view_option'
         verbose_name = "Course View Option"
     
-    def json(self):
+    def json(self, include_id=True):
         """Format the json output that we want for this record
         
+        :param include_id: Whether or not to include the id in the return 
         This should be of the format canvas_id : {options}
         :return: JSON formatted CourseViewOption
         :rtype: Dict
         """
 
```

Comment:
One suggestion I have is to add this line in here: `course_id = canvas_id_to_incremented_id(course_id)`.

For the React rewrite, we send requests using the short course_id.